dcos/jsonitem.py

```diff
@@ -114,6 +122,22 @@ def clean_value(value):
         return value
 
 
+def _validate_value(key, value, pattern):
+    """
+    :param key: Property key
+    :type key: str
+    :param value: Property value to validate
+    :type value: str
+    :param pattern: Validation pattern
+    :type pattern: str
+    :rtype: None
+    """
+
+    if pattern and not re.match(pattern, value):
+        raise DCOSException('Value "{}" does not meet the requirements'
+                            ' for "{}" property'.format(value, key))
```

Comment:
I think all of the error messages below use single quote `''` for printing dynamic string values. Can you double check this? If so can we make it consistent? Change this to use single quote `''`.